index.js

```diff
@@ -144,15 +154,24 @@ module.exports = {
   }
 };
 
-function npmAsset(filePath) {
+function npmAsset(options = {}) {
```

Comment:
Refactoring here required to silence a source maps warning for route-recognizer.